opensrp-chw/build.gradle

```diff
@@ -249,14 +250,15 @@ dependencies {
 //        exclude group: 'id.zelory', module: 'compressor'
 //    }
 
-    implementation('org.smartregister:opensrp-client-chw-referral:1.3.13-NACP-ALPHA-SNAPSHOT@aar') {
+    implementation('org.smartregister:opensrp-client-chw-referral:1.3.13-NACP-LOCAL-ALPHA-SNAPSHOT@aar') {
         transitive = true
         exclude group: 'org.smartregister', module: 'opensrp-client-core'
         exclude group: 'org.smartregister', module: 'opensrp-client-native-form'
         exclude group: 'org.smartregister', module: 'opensrp-client-configurable-views'
         exclude group: 'com.android.support', module: 'appcompat-v7'
         exclude group: 'id.zelory', module: 'compressor'
         exclude group: 'com.rengwuxian.materialedittext', module: 'library'
+        exclude group: 'com.ibm.fhir', module: 'fhir-path'
```

Comment:
@nKataraia - I think this has been fixed so we dont need to exclude fhir-path library and include the version below.